### ISSUE TITLE
CRC Mismatch

### ISSUE THREAD

**Issue by jaredmales** [usbtemp/utmp-cli#3]
my email on this was rejected as spam. 

I've been using your usbtemp probes for about a year.  Everything was working.  But now we are suddenly getting results like:

[]$ sudo ./utmp-cli -s /dev/ttyUSB0
USB Thermometer CLI v1.06 Copyright 2019 usbtemp.com Licensed under MIT licence.
Using serial port: /dev/ttyUSB0
Waiting for response ...
Aug 24 20:11:45 Sensor C: 21.81
[]$ sudo ./utmp-cli -s /dev/ttyUSB0
USB Thermometer CLI v1.06 Copyright 2019 usbtemp.com Licensed under MIT licence.
Using serial port: /dev/ttyUSB0
Waiting for response ...
Error, sensor CRC mismatch!
[]$ sudo ./utmp-cli -s /dev/ttyUSB0
USB Thermometer CLI v1.06 Copyright 2019 usbtemp.com Licensed under MIT licence.
Using serial port: /dev/ttyUSB0
Waiting for response ...
Error, sensor not found!

This is on CentOS 7 using a fresh clone of your utmp-cli.  Seems to work ok under ubuntu 18.

Any ideas?

**Comment by jaka** [comment]
None at the first sign. CentOS might have selinux enabled. I've just ran CentOS 7.8 live from USB, and after compilation works. Does every probe behave the same? Is this USB 3.0 port?

**Comment by jaredmales** [comment]
Thanks for the response.  

Both probes developed this problem at the same time.  They were working for months, then started reporting CRC error.  They were plugged in via a USB 3.0 hub the whole time (when working before, and after they stopped).  Today I found that if they are plugged into a 2.0 port on the main board, they work.

It's likely that other devices were changed on the hub (plugged in, unplugged).
